docs/source/markdown/podmansh.1.md

```diff
@@ -55,10 +57,15 @@ _EOF
 Users inside of this Quadlet are allowed to become root within the user namespace, and able to read/write content in their homedirectory which is mounted from a subdir `data` of the hosts users account.
 
 ```
-sudo cat > /etc/containers/systemd/users/podmansh.container << _EOF
+sudo useradd -s /usr/bin/podmansh confinedu
+grep confinedu /etc/passwd
+confinedu:x:4009:4009::/home/confinedu:/usr/bin/podmansh
```

Comment:
This is evil.

This section (all the man page actually) is written in the "copy and paste this into your terminal" form, vs the "type these commands" form. I despise the copy-paste form, and this is why: the output of the grep is not clearly demarcated.

Suggestion: remove the `sudo`, and add explicit hash prompts to all commands, making it unambiguous that these are commands run as root. (And also making it obvious that the "confinedu" line is output from grep).

Reply:
Fixed